spec/services/stsv1alpha/oas3.fetched/openapi.yaml

```diff
@@ -0,0 +1,114 @@
+openapi: 3.1.0
+info:
+  title: Equinix Security Token Service
+  description: >-
+    Exchange ID tokens for Equinix access tokens according to managed trust
+    relationships. STS is an alpha service and is not generally available to customers.
+  version: 1.0.0

```

Comment:
Might want to use a different version string here, since 1.0.0 implies expectations about production readiness and/or backwards compatibility.  That said, since the package name includes `alpha` and the spec & README say this is alpha, it's OK to keep this as-is; whenever we have a generally available token exchange endpoint, that would end up in a different, non-alpha package anyway.